7_ReleaseNotes.txt

```diff
@@ -18,6 +18,13 @@
   - Recurrence fixes
   - Pseudoprogression fixes
 - Generic GUI bug fixes and improvements 
+- Apply mask functionaity from drawing panel added
+- bvec reorientation functionality added from Utilities
```

Comment:
I believe it should be **BVEC**

Reply:
Done and moved under Utilities category.